src/components/dialogs/create-study-dialog.js

```diff
@@ -167,43 +174,79 @@ export const CreateStudyDialog = ({ open, onClose, providedCase }) => {
     const [triggerReset, setTriggerReset] = React.useState(true);
 
     const [isParamsDisplayed, setIsParamsDisplayed] = useState(false);
-
+    const [isParamsCaseFileDisplayed, setIsParamsCaseFileDisplayed] =
```

Comment:
`isParamsCaseFileDisplayed` is not the same as `isParamsDisplayed`

Reply:
no they not the same each one have different state